res/layout/activity_edit_closed_group.xml

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Since we don't need an empty state view for this screen the `LinearLayout` can just be the root